File: Day 02 - Data Types/solution.java
import java.io.*;
import java.util.*;
import java.text.*;
import java.math.*;
import java.util.regex.*;

public class Solution {
	
    public static void main(String[] args) {
        int i = 4;
        double d = 4.0;
        String s = "HackerRank ";
		
        Scanner scan = new Scanner(System.in);

        /* Declare second integer, double, and String variables. */
        int j = scan.nextInt();
        double dd = scan.nextDouble();
        scan.nextLine();
        String ss = scan.nextLine();
        
        // Note: If you have trouble reading the entire String, please go back and review the Tutorial closely.
        
        /* Print the sum of both integer variables on a new line. */
        System.out.println((i+j)+"\n"+(d+dd)+"\n"+s.concat(ss));
        /* Print the sum of the double variables on a new line. */
		
        /* Concatenate and print the String variables on a new line; 
        	the 's' variable above should be printed first. */
        scan.close();
    }
}

File: Day 02 - Data Types/solution.php
<?php
$handle = fopen ("php://stdin","r");
$i = 4;
$d = 4.0;
$s = "HackerRank ";
// Declare second integer, double, and String variables.
$fp = fopen("php://stdin","r");
$j = (int)readline();
$e = (float)readline();
$t = readline();

// Read and save an integer, double, and String to your variables.

// Print the sum of both integer variables on a new line.
print($i+$j."\n");
// Print the sum of the double variables on a new line.
print(sprintf("%.1f",$e+$d)."\n");
// Concatenate and print the String variables on a new line
// The 's' variable above should be printed first.
print($s.$t);
fclose($handle);
?>